src/network/tcp.py

```diff
@@ -172,13 +187,12 @@ def sendChunk():
                     if Dandelion().hasHash(objHash):
                         continue
                     bigInvList[objHash] = 0
-                    #self.objectsNewToThem[objHash] = time.time()
         objectCount = 0
         payload = b''
         # Now let us start appending all of these hashes together. They will be
         # sent out in a big inv message to our new peer.
-        for hash, storedValue in bigInvList.items():
-            payload += hash
+        for obj_hash, _ in bigInvList.items():
```

Comment:
You've done the opposite in other hunk.

Reply:
I'm lost, sorry, could you be more specific please? There's one other hunk like that in that file:
```
-            sock, addr = pair
+            sock, _ = pair
```
But maybe you mean why did I ignore the violation elsewhere, as in `# pylint: disable=redefined-outer-name`. The answer is because this would change the api of a function. Due to the large number of changes we're making to remove violations we're working on one (or a few) file(s) at a time. There will be some refactorings that will necessarily affect multiple files (e.g. fixing the badly named variables, functions and classes). If you're not referring to either of these, please comment in PR#1271, otherwise, fixed in PR#1271.